eng/pipelines/templates/steps/typespec-ci.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+- name: Folder
+  type: string
+- name: DisplayName
+  type: string
+
+steps:
+- script: |
+    # Log commands before running
+    set -x
+
+    exitcode=0
+
+    cd ${{parameters.Folder}}
```

Comment:
Not sure if it matters much but you could also make this the working directory of the step.

Reply:
We go back to the repo root to run `git` commands, so I'd rather not use this as working directory.  Also, we should be able to use `tsp compile path/to/spec` from repo root once the bugs with that command are fixed.